### PR TITLE
apply ActionResult changes to expo

### PR DESCRIPTION
**Pull request by bangtoven**

### CODE REVIEW

File: react-native/android/build.gradle [hunk 1]
```diff
@@ -70,6 +70,6 @@ repositories {
 dependencies {
   implementation project(':expo-modules-core')
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${getKotlinVersion()}"
-  implementation "com.coinbase:coinbase-wallet-sdk:0.3.0"
+  implementation "com.coinbase:coinbase-wallet-sdk:0.3.2"
```
Comment (bangtoven) [inline react-native/android/build.gradle:73]: not published yet

File: react-native/ios/CoinbaseWalletSDKExpo.podspec [hunk 2]
```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
   s.static_framework = true
 
   s.dependency 'ExpoModulesCore'
-  s.dependency 'CoinbaseWalletSDK', '0.3.0'
+  s.dependency 'CoinbaseWalletSDK', '0.3.2'
```
Comment (bangtoven) [inline react-native/ios/CoinbaseWalletSDKExpo.podspec:19]: not published yet

**Review by bangtoven — COMMENTED** [review]
@vishnumad 🙏

File: react-native/ios/CoinbaseWalletSDKExpo.podspec (hunk 2 above)
Comment (bangtoven) [reply react-native/ios/CoinbaseWalletSDKExpo.podspec:19]: published https://github.com/CocoaPods/Specs/commit/82b683f95cd5ba8217b2e4befda1747d55dc0bd4

File: react-native/android/build.gradle (hunk 1 above)
Comment (bangtoven) [reply react-native/android/build.gradle:73]: https://github.com/coinbase/wallet-mobile-sdk/pull/46

**Review by vishnumad — APPROVED** [review]

**Pull request merged**